src/applications/appeals/995/config/form.js

```diff
@@ -253,8 +263,10 @@ const formConfig = {
         evidenceSummary: {
           title: 'Summary of evidence',
           path: 'supporting-evidence/summary',
-          uiSchema: evidenceSummary.uiSchema,
-          schema: evidenceSummary.schema,
+          CustomPage: EvidenceSummary,
+          CustomPageReview: EvidenceSummaryReview,
+          uiSchema: {},
```

Comment:
`blankUiSchema` includes a `hideOnReview: true` option that hides the page on the review & submit page. The evidence summary page is the _only_ page we want to show on the review & submit page, so that's why this is an empty object.